src/index.html

```diff
@@ -8,9 +8,40 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="images/Search.svg">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="form1"
+      action="#"
+      method="get">
+      <img src="images/Search.svg" alt="search">
+      <label for="search"></label>
+      <input
+        data-qa="keypress"
+        name="search"
+        id="search"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search"
+        >
```

Comment:
```suggestion
      <input
        data-qa="keypress"
        name="search"
        id="search"
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        class="search"
      >
```
check your code style